app/assets/stylesheets/app/session_preview.scss

```diff
@@ -0,0 +1,3 @@
+.session-preview-element__highlighted {
```

Comment:
![highlight](https://user-images.githubusercontent.com/56056/28588991-2d434a62-7174-11e7-8cf9-b0a983852cef.PNG)

If it is this then I'd be tempted to come up with a different name. So maybe we just call it 'highlight' and then have 

```
<div class="highlight">
  <h1 class="title">
  <p class="super-text">

```
Then css would be 

```
.highlight {
  background: $highlight-background-colour;
}
```

then in colours:

`$highlight-background-colour: #5b8126;`